### ISSUE TITLE
Failed

### ISSUE THREAD

**Issue by Shawzborne** [kinejara/myHueOnTV#1]
![screen shot 2015-11-09 at 11 49 26 am](https://cloud.githubusercontent.com/assets/5931778/11039934/fa34c94e-86d7-11e5-9784-b402b6e5407f.png)

**Comment by kinejara** [comment]
I think this is happening since apple disable the support of iOS frameworks in tvOS,
I guess the quick fix for this is to use a beta version of Xcode 7.

**Comment by kinejara** [comment]
@Shawzborne could you send me a better description of the Error ? I may forget to properly link one of the libraries

**Comment by Shawzborne** [comment]
Should I try installing Xcode beta and re run it?

**Comment by kinejara** [comment]
I don't think so, I just asked somebody else to run the project, he was able build and run project using XCode 7.1, what is the version of your XCode ?

**Comment by Shawzborne** [comment]
7.1 (7B91b)

**Comment by ghost** [comment]
Was this ever fixed? on XCode 7.3 with this error
